torch/_refs/__init__.py

```diff
@@ -3404,14 +3404,15 @@ def new_empty_strided(
 @register_decomposition(torch.ops.aten.zeros)
 @out_wrapper()
 def zeros(
-    size: ShapeType,
-    *,
```

Comment:
why did you remove kwarg-only? dtype and following args are kwarg only.

Reply:
Python syntax actually doesn't allow this:  Keyword-only argument separator not allowed after "*parameter"

Coz, arguments after "*parameter" are implicitly kwarg-only...

Reply:
```
@overload
def ones(size: _size, *, out: Optional[Tensor]=None, dtype: Optional[_dtype]=None, layout: Optional[_layout]=None, device: Optional[Union[_device, str, None]]=None, pin_memory: Optional[_bool]=False, requires_grad: Optional[_bool]=False) -> Tensor: ...
@overload
def ones(*size: _int, out: Optional[Tensor]=None, dtype: Optional[_dtype]=None, layout: Optional[_layout]=None, device: Optional[Union[_device, str, None]]=None, pin_memory: Optional[_bool]=False, requires_grad: Optional[_bool]=False) -> Tensor: ...
```

These are the overloads for torch.ones. 
So def(*size, *, ...) is actually invalid

Reply:
ok cool, can you add some sample or reference input to exercise new code path? Apparently tests were not failing previously even though they should have.

Reply:
I added the test cases in opinf db for ones and zeros....
However, the test infra is alway normalizing the inputs to a tuple, so technically it's still not testing the non-tuple size case... 
I guess if we really want to test the non-tuple case, we need to carve out an code path just for this purpose. Do we really want to do this?

Reply:
Yeah ok, we can leave it as is.